packages/front-end/components/Events/EventsPage/utils.ts

```diff
@@ -1,52 +1,76 @@
-import {
-  EventInterface,
-  ExperimentCreatedNotificationEvent,
-  ExperimentDeletedNotificationEvent,
-  ExperimentUpdatedNotificationEvent,
-  FeatureCreatedNotificationEvent,
-  FeatureDeletedNotificationEvent,
-  FeatureUpdatedNotificationEvent,
-  UserLoginNotificationEvent,
-} from "back-end/types/event";
-import { NotificationEvent } from "back-end/src/events/notification-events";
-
-export const getEventText = (
-  event: EventInterface<NotificationEvent>
-): string => {
+import { EventInterface } from "back-end/types/event";
+
+// region Feature
+
+const getTitleForFeatureCreated = ({ id }: { id: string }) =>
+  `The feature ${id} was created`;
```

Comment:
These descriptions should probably not be generated on the front-end like this.  It seems like this would be pretty much the same logic as building a Slack message, or if we add email notification in the future, a subject line.

Feels like we need a centralized way to convert an event into a short human-readable description (or maybe a few different versions with varying verbosity).  We'll also likely need slightly different formatting (e.g. Markdown for Slack, HTML for email, Plain Text for other things).

Reply:
I agree. This PR is mostly re-organizing code. Do you want to add that to the current work or as a follow-up?